surveys/2021-annual-survey/questions.md

```diff
@@ -481,6 +481,26 @@ Select all that apply:
 > particular we can use this information to do cohort analysis on highly "active"
 > community members in comparison to less active community members.
 
+### How often have felt *un*welcome in the Rust community?
+
+Activities:
+
+- *Official* Rust community forums or chats (e.g., users.rust-lang.org, the official Rust Discord, the Rust Zulip, etc.)
```

Comment:
We should explicitly list some forums (esp r/rust) which are not official, since that is not always clear to people (or add that as a separate activity)